docker-compose.yml

```diff
@@ -6,31 +6,26 @@ services:
       context: .
       dockerfile: Dockerfile.template
       args:
-        OS_VERSION: ${OS_VERSION:-2.58.6_rev1}
+        OS_VERSION: ${OS_VERSION:-2.83.18_rev1}
         DEVICE_TYPE: ${DEVICE_TYPE:-genericx86-64-ext}
     privileged: true
     stop_signal: SIGRTMIN+3
     stop_grace_period: 30s
     volumes:
       - /lib/modules:/lib/modules:ro
       - ${CONFIG_JSON:-./config.json}:/var/local/config.json
-      - boot:/mnt/boot
-      - data:/mnt/data
-      - state:/mnt/state
+      - /mnt/boot
+      - /mnt/data
+      - /mnt/state
+      - /sys/fs/cgroup/systemd
     tmpfs:
       - /run
       - /run/lock
       - /tmp
       - /var/lib/journal
-      - /sys/fs/cgroup/systemd
```

Comment:
what's the reason for removing this one?

Reply:
Apparently a left over. Not needed.

Reply:
@robertgzr When I try to reproduce the behaviour without moving /sys/fs/cgroup/systemd into a anonymous volume restarting the containers fails.

Steps to reproduce:
1. docker-compose scale os=2
2. wait until containers have provisioned to test fleet in balena Cloud
3. terminate docker-compose and let containers scale down / stop
4. rerun: docker-compose scale os=2
```
➜  bosic git:(docker-scaled-balenaOS-devices) ✗ $ docker compose up --scale os=2
[+] Running 2/0
 ⠿ Container bosic-os-1  Created                                                                                                                                                                                                                                                                                                                                                                                                    0.0s
 ⠿ Container bosic-os-2  Created                                                                                                                                                                                                                                                                                                                                                                                                    0.0s
Attaching to bosic-os-1, bosic-os-2
bosic-os-2  | Failed to find module 'autofs4'
bosic-os-2  | Failed to create symlink /sys/fs/cgroup/net_cls: File exists
bosic-os-2  | Failed to create symlink /sys/fs/cgroup/net_prio: File exists
bosic-os-2  | Failed to create symlink /sys/fs/cgroup/cpuacct: File exists
bosic-os-2  | Failed to create symlink /sys/fs/cgroup/cpu: File exists
bosic-os-2  |
bosic-os-2  | Welcome to balenaOS 2.83.18+rev1!
bosic-os-2  |
bosic-os-2  | Failed to attach 1 to compat systemd cgroup /docker/46af8594f4ed8d3308f5316293343add112a0d14e6763d34755b1d25b606120f/init.scope: No such file or directory
bosic-os-2  | Failed to open pin file: No such file or directory
bosic-os-2  | Failed to allocate manager object: No such file or directory
bosic-os-2  | [!!!!!!] Failed to allocate manager object.
bosic-os-2  | Exiting PID 1...
bosic-os-1  | Failed to find module 'autofs4'
bosic-os-1  | Failed to create symlink /sys/fs/cgroup/cpuacct: File exists
bosic-os-1  | Failed to create symlink /sys/fs/cgroup/cpu: File exists
bosic-os-1  | Failed to create symlink /sys/fs/cgroup/net_cls: File exists
bosic-os-1  | Failed to create symlink /sys/fs/cgroup/net_prio: File exists
bosic-os-1  |
bosic-os-1  | Welcome to balenaOS 2.83.18+rev1!
bosic-os-1  |
bosic-os-1  | Failed to attach 1 to compat systemd cgroup /docker/ba16c31eecf5ba0bd8aa1f67dd4d429457d57bb6f9144e2cee1e1fb3648fb41a/init.scope: No such file or directory
bosic-os-1  | Failed to open pin file: No such file or directory
bosic-os-1  | Failed to allocate manager object: No such file or directory
bosic-os-1  | [!!!!!!] Failed to allocate manager object.
bosic-os-1  | Exiting PID 1...
bosic-os-2 exited with code 255
bosic-os-1 exited with code 255
```

It feels like the balenaOS in container needs to persist something into the `/sys/fs/cgroup/` and fails to read it back on the second run.